### ISSUE TITLE
Docs: Please describe the parameters of the webhook

### ISSUE THREAD

**Issue by makthis** [replicate/replicate-python#141]
The webhook feature is a great way to deal with long waits.

However, there is no info on what kind of request would it be, what would be the payload, etc.

It would quite helpful.

**Comment by zeke** [comment]
Good idea. I opened a PR here: https://github.com/replicate/replicate-python/pull/187
